src/components/BarChart/BarChart.js

```diff
@@ -1,80 +0,0 @@
-import React from 'react';
```

Comment:
I didn't really delete this, just renamed it to `<CityBarChart>`. But most of it is different.